app/views/content/ways-to-train.md

```diff
@@ -42,208 +42,151 @@ keywords:
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a PGCE or PGDE <span>course</span></a>
-</p>
+In addition to QTS you'll get a PGCE or PGDE (an academic qualification).
 
-$school-direct-fee-funded$
+Courses typically take one year but you may be able to study part-time.
 
-You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+Fees are usually around £9,250 but you can:
 
-You’ll be based in a school but you will not be employed by them.
+* get [funding that you do not have to pay back](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#bursaries-and-scholarships) for certain subjects
+* apply for a [student loan](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#tuition-fee-and-maintenance-loans)
 
-You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+<div class="training-attributes">
+  <h4>University-led training</h4>
+  <div class="badges">
+    <span class="badge pink">Costs £9,250</span>
+    <span class="badge cyan">Funding available</span>
+    <span class="badge purple">Takes 1 year</span>
+    <span class="badge dark-blue">PGCE or PGDE</span>
+  </div>
+</div>
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+### Train as you teach at a school or group of schools
 
-You can also [get funding](/funding-your-training).
+On a 'school direct (fee-funded)' programme you spend less time learning about the theory of teaching than you would in [university-led training](/ways-to-train#go-back-to-university-as-a-postgraduate-student).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+Despite the focus on training as you teach, many courses give you a PGCE or PGDE (an academic qualification) in addition to QTS.
 
-$school-direct-salaried$
+Courses typically take one year but you may be able to study part-time.
 
-You’ll be employed by a school as an unqualified teacher while you work towards your Qualified Teacher Status (QTS).
+Fees are usually around £9,250 but you can:
 
-You’ll earn a salary and do not need to pay any tuition fees.
+* get [funding that you do not have to pay back](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#bursaries-and-scholarships) for certain subjects
+* apply for a [student loan](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#tuition-fee-and-maintenance-loans)  
 
-This route typically requires 3 years of transferable work history.
+<div class="training-attributes">
+  <h4>School direct<br/>(fee-funded)</h4>
```

Comment:
@peteryates what's our general position on using <br> tags in the code? I've never seen any in before